sdk/docker-solana/Dockerfile

```diff
@@ -1,7 +1,35 @@
 FROM debian:buster
 
-# JSON RPC port
+# RPC JSON
 EXPOSE 8899/tcp
+# RPC pubsub
+EXPOSE 8900/tcp
+# entrypoint
+EXPOSE 8001/tcp
+# (future) bank service
+EXPOSE 8901/tcp
+# bank service
+EXPOSE 8902/tcp
+# faucet
+EXPOSE 9900/tcp
+# tvu
+EXPOSE 8000/udp
+# gossip
+EXPOSE 8001/udp
+# tvu_forwards
+EXPOSE 8002/udp
+# tpu
+EXPOSE 8003/udp
+# tpu_forwards
+EXPOSE 8004/udp
+# retransmit
+EXPOSE 8005/udp
+# repair
+EXPOSE 8006/udp
+# serve_repair
+EXPOSE 8007/udp
+# broadcast
+EXPOSE 8008/udp
```

Comment:
These changes are great, devs can use `docker run -P` to easily publish all ports

Reply:
Yeah I hope it'll be useful!  The weird part is that `-P` publishes the ports to random other ports -- from https://docs.docker.com/engine/reference/builder/#expose :

> In this case, if you use -P with docker run, the port will be exposed once for TCP and once for UDP. Remember that -P uses an ephemeral high-ordered host port on the host, so the port will not be the same for TCP and UDP.

But it seems to work nicely if you're using a full Docker-based solution

Reply:
Ah yeah that is unfortunate. This is still nice though!